bundles/binding/org.openhab.binding.zwave/database/products.xml

```diff
@@ -1196,6 +1196,23 @@
 			<Label lang="en">Z-Wave Thermostat</Label>
 			<ConfigFile>rtc/ct100.xml</ConfigFile>
 		</Product>
+		<Product>
```

Comment:
How do you know all three of these codes are for this device? Generally, I would prefer unless you're absolutely sure (ie have tested) that these codes are all associated with this device, that we just use the one(s) you have tested. Otherwise, if there's a mistake it's hard to work out what's wrong if someone later comes along with a device using these codes.